components/layout-wrapper.tsx

```diff
@@ -41,30 +41,54 @@ const LayoutWrapper = ({ children }: Props) => {
             </Link>
             <ul className="relative hidden pt-8 md:flex">
               {leftHeaderNavLinks.map((link) => (
-                <li key={link.title}>
+                <>
```

Comment:
I'm still seeing horizontal scrolling on [the search page](https://tailscale-htjugz306-tailscale.vercel.app/search?q=) at 796px wide.

Reply:
Fixed with [48d9643](https://github.com/tailscale-dev/tailscale-dev/pull/291/commits/48d9643fb74e04b9a3c47e763e2020122994774e). It's a compromise though.